core/bot_framework_adapter.go

```diff
@@ -46,7 +47,10 @@ func NewBotAdapter(settings AdapterSetting) Adapter {
 	if settings.ChannelService == "" {
 		settings.ChannelService = auth.ChannelService
 	}
-	return &BotFrameworkAdapter{settings}
+
+	jwtValidator := auth.NewJwtTokenValidator()
+
+	return &BotFrameworkAdapter{settings, jwtValidator}
```

Comment:
No need to create new variable. Could just return `&BotFrameworkAdapter{settings, auth.NewJwtTokenValidator()}